chef/cookbooks/monasca/attributes/default.rb

```diff
@@ -36,6 +40,12 @@
 
 default[:monasca][:api][:bind_host] = "*"
 
+# agent default service settings
+default[:monasca][:agent][:service_name] = monasca_agent_service_name
```

Comment:
I am thinking if it'b better to go with **:metric_agent** just to match log agent ?